README.md

```diff
@@ -33,10 +33,11 @@ const { init, parse } = require('es-module-lexer');
 })();
 ```
 
-An ES module version is also available from `dist/es-module-lexer.js`:
+An ES module version is also available from `dist/lexer.js`:  
+Note: This version will be automatically used in rollup/es-dev-server/...  
 
 ```js
-import { init, parse } from 'es-module-lexer/dist/es-module-lexer.js';
+import { init, parse } from 'es-module-lexer';
```

Comment:
Strictly speaking, this isn't quite right since this references the CJS version.

If we update the package.json to read:

```js
{
  "main": "./dist/es-module-lexer.cjs",
  "exports": {
    "require": "./dist/es-module-lexer.cjs",
    "import": "./dist/es-module-lexer.js"
  }
}
```

Then Node.js would use the ES module version when modules are in use, and this readme would then be strictly correct.

Let me know if you'd like to add that to the PR, otherwise I will merge it with that in due course.

Reply:
> Strictly speaking, this isn't quite right since this references the CJS version.

in which usage implementation? for rollup and es-dev-server the "module" field in the package.json is all that is needed to prefer this package.

I thought node did the same? if it needs the `exports` then the relevant parts in the package.json would end up to be something like this right?

```
{
  "main": "./dist/lexer.cjs",
  "module": "./dist/lexer.js",
  "exports": {
    "require": "./dist/lexer.cjs",
    "import": "./dist/lexer.js"
  }
}
```

Reply:
Yes, this is for Node.js.

Reply:
(and that is correct)

Reply:
updated the info and added the needed section to package.json 👍 

shall I squash the commits or will you do that anyway on merge?